README.md

```diff
@@ -1,23 +1,36 @@
 # Enarx
 
-## Introduction
-Enarx is an application deployment system enabling applications to run within Trusted Execution Environments (TEEs) without rewriting for particular platforms or SDKs. It handles attestation and delivery into a run-time “Keep” based on WebAssembly, offering developers a wide range of language choices for implementation. Enarx is CPU-architecture independent, enabling the same application code to be deployed across multiple targets, abstracting issues such as cross-compilation and differing attestation mechanisms between hardware vendors. Work is currently underway on AMD SEV and Intel SGX.
-
-We've known for a long time that we need encryption for data at rest and in transit: Enarx helps you do encryption for data as well as algorithms in use.
+## Introduction Enarx is an application deployment system enabling applications
+to run within Trusted Execution Environments (TEEs) without rewriting for
```

Comment:
Was it your intention to have the first part of this paragraph as a header here?

If so, I think the word "Introduction" should be removed. 

If not, I think the words following "Introduction" should be moved into the paragraph section.

Reply:
Good catch. Will correct, thanks.